src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs

```diff
@@ -166,9 +166,12 @@ public override async Task InitializeAsync()
             this.maturedBlockRequester.Start();
 
             // Connect the node to the other federation members.
-            foreach (var federationMemberIp in this.federationGatewaySettings.FederationNodeIpEndPoints)
+            if (this.federationGatewaySettings.FederationNodeIpEndPoints != null)

```

Comment:
Add this after starting a node in integration, as `FederationNodeIpEndPoints` can be null at startup.